sql/moz-fx-data-shared-prod/stripe_derived/invoices_v1/query.sql

```diff
@@ -0,0 +1,22 @@
+SELECT
+  -- limit fields in stripe_derived so as not to expose sensitive data
+  id,
+  billing_reason,
+  customer,
+  status,
+  status_transitions,
+  ARRAY(
+    SELECT
+      STRUCT(
+        line.id,
+        line.amount,
```

Comment:
legal has told us that this should not be visible to the whole company in re:dash, so if we are exposing these values they need to be restricted to a separate data source with access limited to a specific working group.